### ISSUE TITLE
Highlight no-longer-in-sync ManagedOSVersions

### ISSUE THREAD

**Issue by anmazzotti** [rancher/elemental-ui#183]
A new feature (https://github.com/rancher/elemental-operator/pull/750) was added to the elemental-operator to mark out-of-sync ManagedOSVersions.

Two annotations has been added to ManagedOSVersions (which are created from a ManagedOSVersionChannel):

```
elemental.cattle.io/channel-last-sync: "2024-06-12T12:24:59Z"
elemental.cattle.io/channel-no-longer-in-sync: "true"
```

The `channel-no-longer-in-sync` annotation can be used to highlight the `ManagedOSVersion` is deprecated and can be deleted.
This should be visible when listing the ManagedOSVersions from the UI extension.

`ManagedOSVersions` that are `no-longer-in-sync` can also be excluded from being used during ISO/Disk creation from the UI, however the preference if not too hard to implement, is to allow using these versions but also highlight they are deprecated (with an exclamation mark, tooltip or anything when selecting them).

**Comment by aalves08** [comment]
@anmazzotti I believe that what we want is to add a table col in ManagedOSVersions list with in-sync (bool):


![Image](https://github.com/rancher/elemental-ui/assets/97888974/7dddc790-6c26-441b-b357-80e3b28fe901)

And also we should add the label `(deprecated)` to:

![Image](https://github.com/rancher/elemental-ui/assets/97888974/a1c73408-4220-44eb-bef7-bd38b167c825)


That would be the scope, correct @anmazzotti ? 

Question: in what version of `elemental-operator` will this feature be available?

**Comment by anmazzotti** [comment]
Yes that's correct. This will be available in 1.6

Also note that I'm going to add automatic os cleaning, this will be a togglable option when creating the ManagedOSImageChannel: https://github.com/rancher/elemental-operator/issues/759

I'm trying to squeeze this one too. I'll open a new UI issue as soon at it is done, just an heads up for now.

Thank you.

**Comment by juadk** [comment]
Tested with rancher 2.9.0 / operator 1.6.3 / elemental UI 2.0.0-rc1:
![Image](https://github.com/user-attachments/assets/7e6279bc-ae2d-4dcb-b1a1-a225ceb5441d)
